src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/casetransfer/CaseTransferSameCountryServiceTest.java

```diff
@@ -290,6 +290,20 @@ void transferCaseNoCasesFoundThrowsException() {
                 caseDetails, USER_TOKEN));
     }
 
+    @Test
+    void caseTransferShouldTransferSameOfficeWithBfActionAndHearingListed() {
+        String officeCT = TribunalOffice.MANCHESTER.getOfficeName();
+        CaseDetails caseDetails = createCaseDetails(TribunalOffice.MANCHESTER.getOfficeName(), officeCT,
```

Comment:
why are we explicitly doing TribunalOffice.MANCHESTER.getOfficeName() and also officeCT (same thing)?